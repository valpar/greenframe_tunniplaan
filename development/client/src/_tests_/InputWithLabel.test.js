import React from 'react';
import { render, fireEvent, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import InputWithLabel from '../components/UI/Input/InputWithLabel.js';


describe('InputWithLabel', () => {
  it("matches snapshot", () => {
    const tree = renderer.create(<InputWithLabel />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});