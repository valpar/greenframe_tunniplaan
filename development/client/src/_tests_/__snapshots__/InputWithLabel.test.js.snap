// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputWithLabel matches snapshot 1`] = `
<div
  className="relative flex flex-col w-full"
>
  <input
    autoComplete="off"
    className="p-2 w-full border border-borderGray  rounded-none font-normal focus:outline-none focus:border-black undefined"
    name=""
    onChange={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    readOnly={false}
    type="text"
    value=""
  />
</div>
`;
